### ISSUE TITLE
6C spontaneously restarting

### ISSUE THREAD

**Issue by alexpiet** [AllenNeuralDynamics/dynamic-foraging-task#1054]
I noticed this happen a few weeks ago, but it seemed like its stopped. Xinxin reported it happening yesterday. Lets log occurrences here, and if it happens again contact IT.

**Comment by ellahiltonvano** [comment]
Here is the response from IT: 

I've analyzed both MEMORY.DMP files from 11/12 and 11/20. First one included a message as an insufficient memory error. Impacted module was WdFilter which typically comes from a driver error. Driver error blue screens can be very, very difficult to pin-point, however it sounds like a memory ceiling issue, especially given your statement about larger datasets. The second dump file pointed at the nt module, which indicates a kernel failure. The process of cause is stated as "python.exe", with the fault occurring on processor 8.

 

Given public knowledge on voltage issues with Raptor Lake CPUs known to cause some issues under heavy load (W10DT714673 has a i9-14900k), I decided to look at the current bios version:

 

---------------------------------------------------------------------

PS C:\WINDOWS\system32> get-ciminstance -class win32_baseboard -computername "W10DT714673" | format-table Manufacturer, Product, SerialNumber, Version -auto

 

Manufacturer          Product           SerialNumber    Version

------------          -------           ------------    -------

ASUSTeK COMPUTER INC. PRIME Z790-P WIFI 231128883402752 Rev 1.xx

 

 

PS C:\WINDOWS\system32> get-wmiobject -class win32_bios -computername "W10DT714673"

 

 

SMBIOSBIOSVersion : 1604

Manufacturer      : American Megatrends Inc.

Name              : 1604

SerialNumber      : System Serial Number

Version           : ALASKA – 1072009

---------------------------------------------------------------------

 

The BIOS version present on the computer is prior to the release of microcode fixes in newer BIOS versions. This, I can update.

 

So, that being said--
TL;DR—

I’d recommend adding another 64GB of memory to the system to combat the memory insufficiency errors (we can do this, would cost your department $232)
At the same time, I’ll update the BIOS to address known microcode issues with the CPU, as this could be a root cause of some crashes related to the CPU.
 

If you’d like to pursue, let me know when you’d be able to decommission the computer for a day, and I could get this work done!


@alexpiet do you think this is worth doing?

**Comment by alexpiet** [comment]
@ellahiltonvano Add this to the behavior meeting tomorrow. I suspect its worth doing, but:
- Do all the 446 computers have the same BIOS issue?
- Does IT have any idea why this only happens on this computer and not the others which were all purchased at the same time and likely have the same memory and BIOS?

**Comment by ellahiltonvano** [comment]
Matt made these updates this morning. We should monitor this computer and decide if we'd like updates made to the other behavior computers

**Comment by ellahiltonvano** [comment]
6C seems stable. @alexpiet is it worth updating all computers or wait until there is an issue? Matt was going to take a look at the BIOS version on each rig in 446 in the meantime.

**Comment by alexpiet** [comment]
Lets see what Matt says

**Comment by ellahiltonvano** [comment]
Emailed Matt again to follow up on this. Things have seemed stable in terms of this issue
